src/services/playbookService.js

```diff
@@ -0,0 +1,138 @@
+import moment from 'moment'
+import ByPath from 'object-path'
+
+export const IsBootstrapNeeded = (props) => !props.eventType && !props.isFetching
+
+export const BootstrapIfNeeded = (props) => {
+    if(IsBootstrapNeeded(props))
+    {
+        props.dispatch(props.eventTypeActions.fetchEventType(props.eventTypeId))
+    }
+}
+
+export const selectSourceObject = (sourceObjectEnum, event, ticket, eventType, engagement) => {
+    switch(sourceObjectEnum){
+        case 0: return event
+        case 1: return ticket
+        case 2: return eventType
+        case 3: return engagement
+        default: return null //not sure if there's a better behavior for undefined enum
+    }
+}
+
+export const GetComparisonValue = (condition) => {
```

Comment:
This function and the ones below appear to be test helper functions, do they need to exist in the `playbookService.js`?

Reply:
These are business logic functions. The 'testable' versions allow injection of function dependencies so that each can be tested in isolation; the 'testable' version are only meant to be used by testing, the non-testable versions are the actual implementations with the real dependencies pre-injected.